android/src/main/java/com/mapbox/mapboxgl/MapboxMapOptionsSink.java

```diff
@@ -28,4 +28,8 @@
   void setZoomGesturesEnabled(boolean zoomGesturesEnabled);
 
   void setMyLocationEnabled(boolean myLocationEnabled);
+  
+  void setMyLocationTrackingMode(int myLocationTrackingMode);
+
+  void setMyLocationVerticalAlignment(int myLocationVerticalAlignment);
```

Comment:
not actionable atm but we might need to look into either using enums or using @StringDef/@IntDef for type safety

Reply:
Since it's on the Java side, I thought it's enough to rely on [this](https://github.com/tobrun/flutter-mapbox-gl/pull/21/files/378256116136686c3df14b688c9f8e60da96bf2a#diff-5def2a7288f868311179d4a318f1183dR592) array.